.github/workflows/e2etest-run-tests.yml

```diff
@@ -109,14 +110,10 @@ jobs:
 
       - name: Run E2E tests
         env:
-          E2E_OSCONFIG_IOTHUB_CONNSTR: ${{ steps.hub-identity.outputs.iothubowner_connection_string }}
-          E2E_OSCONFIG_DEVICE_ID: ${{ steps.get-test-data.outputs.device_id }}
-          E2E_OSCONFIG_TWIN_TIMEOUT: ${{ secrets.TWIN_TIMEOUT }}
-          E2E_OSCONFIG_DISTRIBUTION_NAME: ${{ matrix.distroName }}
           DOTNET_CLI_HOME: /tmp
         working-directory: ./src/tests/e2etest
         run: |
-          dotnet test ${{ steps.get-test-data.outputs.test_filter }} --logger "trx;LogFileName=test-results-${{ matrix.distroName }}.trx"
+          sudo E2E_OSCONFIG_IOTHUB_CONNSTR="${{ steps.hub-identity.outputs.iothubowner_connection_string }}" E2E_OSCONFIG_DEVICE_ID="${{ steps.get-test-data.outputs.device_id }}" E2E_OSCONFIG_DISTRIBUTION_NAME="${{ matrix.distroName }}" E2E_OSCONFIG_TWIN_TIMEOUT=${{ secrets.TWIN_TIMEOUT }} dotnet test ${{ steps.get-test-data.outputs.test_filter }} --logger "trx;LogFileName=test-results-${{ matrix.distroName }}.trx;verbosity=detailed"
```

Comment:
Just curious, why move these environment variables out of `env` ?

Reply:
because env only give the current users environment variables, we require them with our privileged user now because we need write access to the local desired json.